core/point/point.go

```diff
@@ -12,6 +12,12 @@ type Point struct {
 	y int64
 }
 
+// Coord is a float point for representing points in space
+type Coord struct {
```

Comment:
How about we move this to core/point/coord.go? Would make it a bit clearer, I think.